src/ui/Windows/MainMenu.cpp

```diff
@@ -1431,6 +1443,17 @@ void DboxMain::OnContextMenu(CWnd * /* pWnd */, CPoint screen)
       case CItemData::ET_ALIAS:
       case CItemData::ET_SHORTCUT:
         pPopup->RemoveMenu(ID_MENUITEM_CREATESHORTCUT, MF_BYCOMMAND);
+        if (m_bFilterActive) {
```

Comment:
If a filter is active, the base entry may not be in the current Tree/List view and so can't be displayed.